packages/engine/errors/7053.md

```diff
@@ -0,0 +1,20 @@
+---
+original: "Element implicitly has an 'any' type because expression of type '{0}' can't be used to index type '{1}'."
+excerpt: "You are trying to access property of an object with the incorrect type. "
```

Comment:
This excerpt isn't correct - it's specifically about the quality of index type `{0}` being not assignable to the index of `{1}`.

I also don't think the body is accurate based on my experience with this error.

Reply:
Could you provide a TSPlayground of this error and we can work from there?

Reply:
Hi @mattpocock, Thanks for the feedback
here is the ts playground link for the error
https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgPICMBWyDeAoZQ5EOAWwgC5kBnMKUAcwG48BfPBAexFuU6yoZsAXlwEiJclQDkAFQCeABwjUE9RWGls8MAK4gEYYN2QMIYAHJkIACkmViu0umgBKKgDdOwACZiiyFw8YHxYVuTIovyYANr2ALpsQA